### ISSUE TITLE
Getting help if you can't sign in.

### ISSUE THREAD

**Issue by birdsarah** [mozilla-mobile/mozilla-vpn-client#702]
How do we make it easier for users to get support if they can't sign in?

If you're on a client and can't sign in there's a "Get Help" link in the top right. 

![2B3990B2-A6EE-4201-A6E5-B60104EC0C5F](https://user-images.githubusercontent.com/1796208/111503787-11586200-8715-11eb-8d85-3f272e4a2d5d.png)

One you click it, there's a "Contact Us" link. 

![6E72F928-C81F-49DA-B7BD-9D5CAD326930](https://user-images.githubusercontent.com/1796208/111503913-2df49a00-8715-11eb-805e-426ed228d0b8.png)

But it takes you to accounts.firefox.com home page. 

![30BF6854-591A-4ADB-87D2-CCE24ACA244C](https://user-images.githubusercontent.com/1796208/111504042-482e7800-8715-11eb-9c24-81f5aabe1003.png)

Which if you're struggling to sign-in is probably quite frustrating.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-121)

**Comment by birdsarah** [comment]
cc @jr-johnson-support

**Comment by bakulf** [comment]
cc @jess-cook03

**Comment by jess-cook03** [comment]
@bakulf @birdsarah 
This is a great flag. To my understanding with current 'contact support' options (from a past and present life working on support experience at mozilla heh.) ...... The contact support option as it lives today is only available to paying customers.... meaning you have to already be a subscriber in order to contact support.

So that is why it is prompting user to sign in/up for them to contact us.

I can follow up with JR on this and future plans for premium support and to confirm what I said above.

An alternative for users to get support is the 'Help center' in this menu which brings them directly to support.mozilla.org where they can self help or ask a question in our forums (which they also need a login to do that as well).... so its a catch 22.

The only other thought I have on this is if we were able to detect if a user is a subscriber or not ? So when they select 'get help' we either directly link them support.mozilla.org (if they are not a subscriber) and we keep treatment as is for those users who are subscribers.

**Comment by jr-johnson-support** [comment]
@jess-cook03 We should chat about revisiting this flow. An alternate use case is for existing fxa/subscriptions account holders who lose/forget their password or are locked out of 2FA.

**Comment by jess-cook03** [comment]
@jr-johnson-support Okay -- yes let's have a conversation here around this. I'm adding this into my baclog list of items. Marking as priority 2 for now -- will assess if this is something we can triage for q2.

**Comment by rbillings** [comment]
This needs a discussion of the correct flows.

**Comment by jess-cook03** [comment]
UX effort is Small-Medium depending on business decision and support decisions on this.
Quick UX solution is -- We could direct users to the support contact form in this instance and give premium support to essentially a larger group of users... not gated by a login. This would mean.... we offer premium support to not just users who have an account. But support would need to make that decision and understand if they have the capacity to service that kind of UX change.
I do not see this as blocked by UX right now. I would say if this plans to be in v2.3 larger discussions need to be had before a UX solution can and should be implemented.
cc @dianahorn @brianyee and @jr-johnson-support

**Comment by bfyee** [comment]
We need to support this scenario - as we have seen from Mullvad DOS issues, our paying customers are unable to sign into the client.  As a premium product,  being unable to contact support will lead to cancellations.  Understand there will be unsubscribed customers contacting support, but that is a tradeoff for helping our paying customers

**Comment by jess-cook03** [comment]
@jr-johnson-support @bfyee to accomplish this for v2.4 for **_post purchase_** 

To provide proper design documentation.... I am trying to understand more about what we need to do from a UX/UI perspective if a user is struggling to sign into their FXA account.

Based on the decision/clarifying factor that we discussed in our v2.4 meeting we will tackle is issue from a post purchase perspective... meaning they already have an account by now and I would imagine they are having issues signing in because they forgot their credentials etc... we are then actually looking to add 'get help' links into FXA flow which actually would not be covered in this repo by us. It would then need to be passed over to subplat and that team......

However what we can do which we talked about before from our end is not 'block' the contact support form with the need to sign in....which there would be no new designs on our end we would just link/direct the users to the form.... however from a support/backend perspective is that possible to have users submit help/contact forms without being able to tie them to a FXA account... will that impact logs and data we can get to help the user? I don't know, maybe its fine?

With funnel optimization in the subscription flow (pre purchase) -- we are already thinking of adding a get help link in there as well, so essentially again that is a subplat task to get those links in there where we want them from UX perspective.

Where I am getting at is where/how to we want to direct users and when....?
